docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "MissingEscapingInFormatting",
+        "summary": "String literals passed to internal functions where not properly escaped.",
+        "description": "Passing a string with a backslash to a builtin function would lead to an ICE as the string would be passed on to the IR codegen without being properly escaped.",
```

Comment:
Please do not use abbreviations in bug descriptions.

Reply:
Also, the problem is not the internal error - an internal error would not be a bug list entry.

Did you try this test with e.g. `"\\\\"`?

Reply:
I tried and it worked before. It must be a certain number of backslashes. I adjusted the description